ch2/fizzBuzz.js

```diff
@@ -0,0 +1,8 @@
+for(let i = 1; i <= 100; i++) {
+  console.log(
+    !(i % 3) && !(i % 5) ? "FizzBuzz" :
+    !(i % 3) ? "Fizz" :
+    !(i % 5) ? "Buzz" :
+    i
```

Comment:
yea this works, though maybe more terse than optimal since chained ternaries like this can become hard to read. i was more referring to removing the separate case for the string "FizzBuzz", ie:

```js
for(let i = 1; i <= 100; i++) {
  let output = "";
  if (output % 3) output += "Fizz";
  if (output % 5) output += "Buzz";
  console.log(output === "" ? i :  output);
}
```

no need to change, but moving forward, try to avoid chained ternary expressions!

Reply:
Well that’s pretty clever. Ok noted. lol you said more terse. This did achieve that particular objective.